### PR TITLE
Angular Getting Started Updates

### PR DESCRIPTION
**Pull request by wlee221**
_Issue #, if available:_

_Description of changes:_ Updates angular getting started to use latest `@aws-amplify/ui-angular`, as well as some callouts specific to Angular 12+.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/start/getting-started/angular/setup.mdx [hunk 1]
```diff
@@ -115,8 +123,8 @@ import { AppComponent } from './app.component';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    /* configure app with AmplifyUIAngularModule */
-    AmplifyUIAngularModule
+    /* configure App with AmplifyUIAngularModule */
```
Comment (ErikCH) [inline src/fragments/start/getting-started/angular/setup.mdx:126]: Should this comment still be here or reference the new `AmplifyAuthenticatorModule?

Comment (wlee221) [reply src/fragments/start/getting-started/angular/setup.mdx:126]: You're right!
```suggestion
    /* configure App with AmplifyAuthenticatorModule */
```

**Review by ErikCH — APPROVED** [review]
👍

**Comment by wlee221** [comment]
Additional updates to be made (based on convo with @chrisbonifacio: [WIP]

- In [#connect-frontend-to-api](https://docs.amplify.aws/start/getting-started/data-model/q/integration/angular/#connect-frontend-to-api), “Next, enable the Angular forms modules in src/app/app.module.ts…” should be mentioned before customer sets up forms.
-

**Review by ashika01 — APPROVED** [review]
Looks good 👍

File: src/fragments/start/getting-started/angular/setup.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ cd amplify-app
 
 ### Angular 6+ Support
 
-Currently, the newest versions of Angular (6+) do not include shims for 'global' or 'process' as provided in previous versions. Add the following to your `src/polyfills.ts` file to recreate them:
```
Comment (wlee221) [inline src/fragments/start/getting-started/angular/setup.mdx:16]: Not so new anymore ... 😅

**Review by offlineprogrammer — APPROVED** [review]

**Pull request merged**